Mathlib/Data/List/Sort.lean

```diff
@@ -191,6 +190,86 @@ alias ⟨_, _root_.Monotone.ofFn_sorted⟩ := sorted_le_ofFn_iff
 
 end Monotone
 
+end List
+
+open List
+
+namespace StrictMono
+
+variable {α β : Type*} [LinearOrder α] [Preorder β] {f : α → β} {l : List α}
+
+theorem sorted_le_listMap (hf : StrictMono f) :
```

Comment:
I'd maybe expected `List.sorted_le_map_iff`, but not strongly.

Reply:
It isn't clear how to use this scheme with `StrictMono`/`StrictAnti`/`OrderEmbedding`.